src/olympia/migrations/1037-adds-discovery-item-indexes.sql

```diff
@@ -0,0 +1,4 @@
+CREATE INDEX `discovery_discoveryitem_position`
+    ON `discovery_discoveryitem` (`position`);
+CREATE INDEX `discovery_discoveryitem_position_china`
+    ON `discovery_discoveryitem` (`position_china`);
```

Comment:
Can you add these custom indexes to `Meta.indexes` (https://docs.djangoproject.com/en/1.11/ref/models/options/#django.db.models.Options.indexes) please, now that we are able to do so? We need to do that for all models but should already start for new ones :)

Reply:
They are already in the model as `db_index=True` - the sql was missing :)

Reply:
they are custom index names though, they're not how django would name them.

I may be thinking a bit too far but this is related to easing our way towards django-migrations :)

Reply:
Oh, I used django-extensions `sqldiff` and thought it would give me django names. I definitely wanted that...

Reply:
Fixed in 9e954cc466ee06615277d30e1624327a3fd85622